athena/templates/tdr_cloudtrail_action_on_date.sql.tpl

```diff
@@ -0,0 +1,11 @@
+SELECT eventname,
+       requestparameters,
+       useridentity,
+       readonly
+FROM tdr_cloudtrail_logs
+WHERE timestamp = '2020/10/13'
```

Comment:
I think so. They're meant as placeholder values. Because there's about 3 different formats a date argument can be in, I think it's easier if you can see an example in the query and you can change it as needed.